### PR TITLE
fix(mnq): product split

### PR DESCRIPTION
**Pull request by RoRoJ**
Split Messaging and Queuing into three distinct products
- NATS
- Queues
- Topics and Events

### CODE REVIEW

File: serverless/messaging/how-to/manage-credentials.mdx [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
+3. Use the drop-down menu to select the [region](/serverless/messaging/concepts/#region) containing the credentials you want to revoke.
 
-3. Click **SQS** or **SNS** depending on which service you want to revoke credentials for.
-
-4. In the **Credentials** tab, click the <Icon name="more" /> button next to the credentials you want to revoke and select **Revoke**.
+4. Click the <Icon name="more" /> button next to the credentials you want to revoke and select **Revoke**.
 
     A pop-up displays, warning you that this action is irreversible and the key will no longer be able to access your service.
 
 4. Click **Revoke credentials**.
```
Comment (nerda-codes) [inline serverless/messaging/how-to/manage-credentials.mdx:58]:
```suggestion
5. Click **Revoke credentials**.
```

File: serverless/messaging/how-to/monitor-mnq-cockpit.mdx [hunk 2]
```diff
@@ -27,29 +26,26 @@ These steps are described below.
 
 - A Scaleway account logged into the [console](https://console.scaleway.com)
 - [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
-- A [NATS account or have activated SQS/SNS](/serverless/messaging/how-to/get-started/)
+- At least one [NATS account](/serverless/messaging/how-to/get-started/), [queue](TODO), or [topic](TODO)
```
Comment (nerda-codes) [inline serverless/messaging/how-to/monitor-mnq-cockpit.mdx:29]: reminder

File: serverless/messaging/reference-content/sns-overview.mdx [hunk 3]
```diff
@@ -1,63 +1,62 @@
...
+SNS is a managed messaging service developed by AWS and leveraged by Scaleway Topics and Events.
 
-SNS (Simple Notification Service) was released in 2010. It enables users to interconnect their applications and systems using a [publish/subscribe](/serverless/messaging/concepts#publishsubscribe) pattern. Many publishers can send messages to [topics](/serverless/messaging/concepts#topicexchange), while consumers, e.g. applications, can subscribe to topics to receive all (or a subset) of messages sent to this topic. Scaleway Messaging and Queuing makes it easy to migrate your SNS-connected applications by providing a dedicated protocol gateway.
+SNS (Simple Notification Service) was released in 2010. It enables users to interconnect their applications and systems using a [publish/subscribe](/serverless/messaging/concepts#publishsubscribe) pattern. Many publishers can send messages to [topics](/serverless/messaging/concepts#topicexchange), while consumers, e.g. applications, can subscribe to topics to receive all (or a subset) of messages sent to this topic. Scaleway Topcs and Events makes it easy to migrate your SNS-connected applications by providing a dedicated protocol gateway.
```
Comment (nerda-codes) [inline serverless/messaging/reference-content/sns-overview.mdx:24]:
```suggestion
SNS (Simple Notification Service) was released in 2010. It enables users to interconnect their applications and systems using a [publish/subscribe](/serverless/messaging/concepts#publishsubscribe) pattern. Many publishers can send messages to [topics](/serverless/messaging/concepts#topicexchange), while consumers, e.g. applications, can subscribe to topics to receive all (or a subset) of messages sent to this topic. Scaleway Topics and Events makes it easy to migrate your SNS-connected applications by providing a dedicated protocol gateway.
```

File: tutorials/create-serverless-scraping/index.mdx [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 dates:
-  validation: 2024-09-16
+  validation: 2024-10-21
   posted: 2023-12-08
 ---
 
 In this tutorial, we show how to set up a simple application that reads [Hacker News](https://news.ycombinator.com) and processes the articles it finds there asynchronously. To do so, we  use Scaleway serverless products and deploy two functions:
- - A producer function, activated by a recurrent cron trigger, that scrapes HackerNews for articles published in the last 15 minutes and pushes the title and URL of the articles to an SQS queue created with Scaleway Messaging and Queuing.
- - A consumer function, triggered by each new message on the SQS queue, that consumes messages published to the queue, scrapes some data from the linked article, and then writes the data into a Scaleway Managed Database.
+ - A producer function, activated by a recurrent cron trigger, that scrapes HackerNews for articles published in the last 15 minutes and pushes the title and URL of the articles to an queue created with Scaleway Queues.
```
Comment (nerda-codes) [inline tutorials/create-serverless-scraping/index.mdx:19]:
```suggestion
 - A producer function, activated by a recurrent cron trigger, that scrapes HackerNews for articles published in the last 15 minutes and pushes the title and URL of the articles to a queue created with Scaleway Queues.
```

File: tutorials/transform-bucket-images-triggers-functions-deploy/index.mdx [hunk 5]
```diff
@@ -25,7 +25,7 @@ In the [previous tutorial](/tutorials/transform-bucket-images-triggers-functions
 - A Scaleway [Project](/identity-and-access-management/organizations-and-projects/quickstart/#project) dedicated to this tutorial
 - A source [bucket](/storage/object/how-to/create-a-bucket/) containing images
 - An empty destination bucket
-- An [SQS queue](/serverless/messaging/quickstart/#quickstart-for-sqs) with credentials
+- An [Scaleway queue](/serverless/messaging/how-to/create-manage-queues) with credentials
```
Comment (nerda-codes) [inline tutorials/transform-bucket-images-triggers-functions-deploy/index.mdx:28]:
```suggestion
- A [Scaleway queue](/serverless/messaging/how-to/create-manage-queues) with credentials
```

File: faq/messaging-and-queuing.mdx [hunk 6]
```diff
@@ -1,65 +1,46 @@
...
-The new version of Messaging and Queuing is designed to provide a better and more secure experience, as well as preparing Messaging and Queuing for further integration into the Scaleway ecosystem.
-
-## Can I still keep using the old v1alpha1 API?
+- Scaleway Queues queues
 
-The v1alpha1 API has now been deprecated, and all resources still existing in it deleted.
+For more details on supported and unsupported Topics and Events features, see our [dedicated page](/serverless/messaging/reference-content/sns-support/).
 
-You must migrate by recreating your SQS and/or SNS resources on the new v1beta1 API (or via the console). NATS resources have been automatically migrated.
+## Can I configure Scalewat NATS via Terraform?
```
Comment (bene2k1) [inline faq/messaging-and-queuing.mdx:34]:
```suggestion
## Can I configure Scaleway NATS via Terraform?
```

File: faq/messaging-and-queuing.mdx [hunk 7]
```diff
@@ -1,65 +1,46 @@
...
 
-- [Messaging and Queuing: SQS v1beta1 API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/sqs-api/)
-- [Messaging and Queuing: SNS v1beta1 API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/sns-api/)
-- [Messaging and Queuing: NATS v1beta1 API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/nats-api/)
+Yes, check out our [tutorial](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/guides/mnq_with_nats_terraform_provider) on how to configure with the Terraform [NATS Jetstream provider](https://registry.terraform.io/providers/nats-io/jetstream/latest/docs).
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply faq/messaging-and-queuing.mdx:36]:
```suggestion
Yes, check out our [tutorial](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/guides/mnq_with_nats_terraform_provider) on how to configure Scaleway NATS with the Terraform [NATS Jetstream provider](https://registry.terraform.io/providers/nats-io/jetstream/latest/docs).
```

File: faq/messaging-and-queuing.mdx [hunk 8]
```diff
@@ -1,65 +1,46 @@
...
+## How are NATS, Queues, and Topics and Events billed?
 
-Billing for NATS and SQS is based on:
+Billing is based on:
 
 - For NATS: [Stream volume](/serverless/messaging/concepts/#stream-volume) (the total sum of each message's size going through the stream) and [stream persistence](/serverless/messaging/concepts/#stream-persistence) (the total amount of data stored * duration)
-- For SQS: [Queue volume](/serverless/messaging/concepts/#queue-volume) (the total sum of each message's size going through the queue)
-- For SNS : [Topic Volume](/serverless/messaging/concepts/#queue-volume)(the total sum of each message's size going out from the topic to the subscriptions)
+- For Queues: [Queue volume](/serverless/messaging/concepts/#queue-volume) (the total sum of each message's size going through the queue)
+- For Topics and Events : [Topic Volume](/serverless/messaging/concepts/#queue-volume)(the total sum of each message's size going out from the topic to the subscriptions)
```
Comment (bene2k1) [inline faq/messaging-and-queuing.mdx:44]:
```suggestion
- For Topics and Events: [Topic Volume](/serverless/messaging/concepts/#queue-volume)(the total sum of each message's size going out from the topic to the subscriptions)
```

File: serverless/messaging/concepts.mdx [hunk 9]
```diff
@@ -43,92 +43,79 @@ In a [topic-based](#topic-based) system, where topics handle the logic, filterin
...
 
 ## Message retention period
 
-The message retention period is a setting that can be configured for an SQS queue. It represents the length of time (in seconds) that messages are kept in the queue before being deleted. Setting a longer message retention period allows for a longer interval between a message being sent and it being received. Read more in our dedicated documentation on [creating queues](/serverless/messaging/how-to/create-manage-queues/).
+The message retention period is a setting that can be configured for a  queue. It represents the length of time (in seconds) that messages are kept in the queue before being deleted. Setting a longer message retention period allows for a longer interval between a message being sent and it being received. Read more in our dedicated documentation on [creating queues](/serverless/messaging/how-to/create-manage-queues/).
```
Comment (bene2k1) [inline serverless/messaging/concepts.mdx:54]:
```suggestion
The message retention period is a setting that can be configured for a queue. It represents the length of time (in seconds) that messages are kept in the queue before being deleted. Setting a longer message retention period allows for a longer interval between a message being sent and it being received. Read more in our dedicated documentation on [creating queues](/serverless/messaging/how-to/create-manage-queues/).
```

File: serverless/messaging/how-to/create-manage-subscriptions.mdx [hunk 10]
```diff
@@ -23,24 +23,23 @@ This page shows how to create and manage SNS [subscriptions](/serverless/messagi
...
+3. Click the name of the topic you want to create a subscription for.
 
-4. Click **create subscription**. The subscription creation wizard displays:
+4. Click **Create Subscription**. The subscription creation wizard displays.
+    <Lightbox src="scaleway-topics-create-subs.webp" alt="In the Scaleway console, the Create a Subscription screen displays. The different sections of the screen are described in the steps below" />
 
-    <Lightbox src="scaleway-create-subscription.webp" alt="" />
-
-5. Verify the parameters of the topic that you are creating a subscription for.
+5. Verify the parameters of the topic that you are creating a subscription for (name, type and).
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply serverless/messaging/how-to/create-manage-subscriptions.mdx:35]:
```suggestion
5. Verify the parameters of the topic that you are creating a subscription for (name and type).
```

File: serverless/messaging/quickstart.mdx [hunk 11]
```diff
@@ -1,216 +1,243 @@
...
+3. Click the name of the topic you want to create a subscription for.
 
-4. Click **create subscription**. The subscription creation wizard displays:
+4. Click **Create Subscription**. The subscription creation wizard displays.
+    <Lightbox src="scaleway-topics-create-subs.webp" alt="In the Scaleway console, the Create a Subscription screen displays. The different sections of the screen are described in the steps below" />
 
-    <Lightbox src="scaleway-create-subscription.webp" alt="" />
-
-5. Verify the parameters of the topic that you are creating a subscription for.
+5. Verify the parameters of the topic that you are creating a subscription for (name, type and).
```
Comment (bene2k1) [inline serverless/messaging/quickstart.mdx:233]:
```suggestion
5. Verify the parameters of the topic that you are creating a subscription for (name and type).
```

**Review by bene2k1 — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**